### DISCUSSION TITLE
Handling of a range of wiki functionality

### DISCUSSION THREAD

**Discussion by realpobblebonk** [WikiTransformationProject/wikitraccs-releases#37] [Q&A]
Just wanted to check how the tool handles:

- Wiki Page attachments
- Cross linking between Spaces
- Do you have widgets for auto-surfacing of Tables Of Contents
- We also use JIRA so wonder how it handles these embedded links

**Comment by heinrich-ulbricht** [comment]
@realpobblebonk Hi, sure!

- page attachments are migrated to SharePoint and put into the right folder for the SharePoint page - here are some more details: https://www.wikitransformationproject.com/docs/features/#page-attachments-and-external-images
- WikiTraccs rewrites links to point to the right page in SharePoint; links between spaces are handled as well, if you configure the space-to-site-mapping before starting the migration (this section has more info about links: https://www.wikitransformationproject.com/docs/features/#page-content); looking at it right now it seems the space to site mapping topic needs some love in the docs, I'll update those in the coming week
- [WikiPakk](https://www.wikitransformationproject.com/docs/wikipakk-reference) is destined to bring Confluence-like features to SharePoint to make the transition for users as smooth as possible; it covers the page tree so far, Table of Contents not (yet)
- when Confluence and Jira are connected then links to single Jira issues will be converted to regular web links on the SharePoint page; here is some info about this: https://www.wikitransformationproject.com/docs/features/#jira-issue-links

Does this answer your questions?

**Comment by realpobblebonk** [comment]
Yes, You have helped lots.
   
  
   Unfortunately our organization has a laundry list of things to migrate to the cloud and confluence and JIRA are low on the priority list.
   
  
    
   
  
   However I very interested in keeping up to date as you add functionality.
   
  
    
   
  
   Mat
   
   
   
    On 04/21/2023 12:24 AM PDT Heinrich Ulbricht ***@***.***> wrote:
    
   
     
    
   
     
    
     
   @realpobblebonk Hi, sure! 
    
    page attachments are migrated to SharePoint and put into the right folder for the SharePoint page - here are some more details: https://www.wikitransformationproject.com/docs/features/#page-attachments-and-external-images 
    WikiTraccs rewrites links to point to the right page in SharePoint; links between spaces are handled as well, if you configure the space-to-site-mapping before starting the migration (this section has more info about links: https://www.wikitransformationproject.com/docs/features/#page-content) 
    WikiPakk is destined to bring Confluence-like features to SharePoint to make the transition for users as smooth as possible; it covers the page tree so far, Table of Contents not (yet) 
    when Confluence and Jira are connected then links to single Jira issues will be converted to regular web links on the SharePoint page; here is some info about this: https://www.wikitransformationproject.com/docs/features/#jira-issue-links 
    
   Does this help?
